src/MessageHandler/Security/SendScanResultHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\MessageHandler\Security;
+
+use Buddy\Repman\Message\Security\SendScanResult;
+use Buddy\Repman\Query\User\OrganizationQuery;
+use Buddy\Repman\Service\Mailer;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+final class SendScanResultHandler implements MessageHandlerInterface
+{
+    private Mailer $mailer;
+    private OrganizationQuery $organizationQuery;
+
+    public function __construct(Mailer $mailer, OrganizationQuery $organizationQuery)
+    {
+        $this->mailer = $mailer;
+        $this->organizationQuery = $organizationQuery;
+    }
+
+    public function __invoke(SendScanResult $message): void
+    {
+        $organization = $this->organizationQuery
+            ->getByAlias($message->organizationAlias())
+            ->getOrElseThrow(new \RuntimeException('Organization not found'));
```

Comment:
we won't do anything about it anyway, it may be better to just ignore this mistake in this case and just return